source/conf.py

```diff
@@ -38,6 +38,7 @@
     "gobject": ("https://lazka.github.io/pgi-docs/GObject-2.0", None),
     "gio": ("https://lazka.github.io/pgi-docs/Gio-2.0", None),
     "gtk": ("https://lazka.github.io/pgi-docs/Gtk-3.0", None),
+    "pygobject": ("https://pygobject.readthedocs.io/en/latest", None),
```

Comment:
Why do you need to reference the pygobject documentation, it doesn't seem to contain API docs?

Reply:
It does have some in https://pygobject.readthedocs.io/en/latest/guide/api/index.html.

Here it is providing:

```
:func:`GObject.Property`
```